interface/forms_admin/forms_admin.php

```diff
@@ -46,195 +46,217 @@
-        } elseif ($registry['state'] == "0") {
-            echo "<td bgcolor='#FFCCCC' width='10%'><a class='link_submit' href='./forms_admin.php?id=" . attr_url($registry['id']) . "&method=enable&csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "'>" . xlt('disabled') . "</a>";
-        } else {
-            echo "<td bgcolor='#CCFFCC' width='10%'><a class='link_submit' href='./forms_admin.php?id=" . attr_url($registry['id']) . "&method=disable&csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "'>" . xlt('enabled') . "</a>";
-        }
-        ?></td>
-        <td bgcolor="<?php echo attr($color); ?>" width="10%">
-      <span class='text'><?php
-        if ($registry['unpackaged']) {
-            echo xlt('PHP extracted');
-        } else {
-            echo xlt('PHP compressed');
-        }
-        ?></span>
-        </td>
-        <td bgcolor="<?php echo attr($color); ?>" width="10%">
-        <?php
-        if ($registry['sql_run']) {
-            echo "<span class='text'>" . xlt('DB installed') . "</span>";
-        } else {
-            echo "<a class='link_submit' href='./forms_admin.php?id=" . attr_url($registry['id']) . "&method=install_db&csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "'>" . xlt('install DB') . "</a>";
-        }
-        ?>
-        </td>
-        <?php
-          echo "<td><input type='text' size='4'  name='priority_" . attr($registry['id']) . "' value='" . attr($priority_category['priority']) . "'></td>";
-          echo "<td><input type='text' size='10' name='category_" . attr($registry['id']) . "' value='" . attr($priority_category['category']) . "'></td>";
-          echo "<td><input type='text' size='10' name='nickname_" . attr($registry['id']) . "' value='" . attr($priority_category['nickname']) . "'></td>";
-          echo "<td>";
-          echo "<select name='aco_spec_" . attr($registry['id']) . "'>";
-          echo "<option value=''></option>";
-          echo AclExtended::genAcoHtmlOptions($priority_category['aco_spec']);
-          echo "</select>";
-          echo "</td>";
-        ?>
-      </tr>
-        <?php
-        if ($color=="#CCCCCC") {
-            $color="#999999";
-        } else {
-            $color="#CCCCCC";
-        }
-    } //end of foreach
-}
-?>
-</table>
-<hr>
+            <?php //REGISTERED SECTION ?>
+            <span class="font-weight-bold"><?php echo xlt('Registered');?></span>
+            <form method="post" action ='./forms_admin.php'>
+                <span class="font-italic">
+                    <?php echo xlt('click here to update priority, category, nickname and access control settings'); ?>
+                </span>
+                <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                <input class="btn btn-primary" type='submit' name='update' value='<?php echo xla('Update'); ?>'>
```

Comment:
It seems like this should be `Save` instead of `Update`, since it is saving stuff.